skyfield/data/gravitational_parameters.py

```diff
@@ -82,3 +82,6 @@
     2000511: 2.3312860000000000E+00,
     2000704: 2.3573170000000001E+00,
     }
+
+non_barycenters = [id for id in GM_dict.keys() if id not in range(1, 10)]
+GM_dict[0] = sum([GM_dict[id] for id in non_barycenters])
```

Comment:
Let's see if we can avoid introducing import-time side effects, like calculations. Try adding a little script in `design/` that computes this number and prints its `repr()`, then simply paste the result into this file. No point in burning the world’s fossil fuels recomputing exactly the same number every time someone imports this file, right? :wink:

(Oh: and, does this add the exact same masses that are listed in `gm_Horizons.pck`? And how does the resulting mass compare to the one used by HORIZONS?)

Reply:
I didn't add up all of the masses that were listed in the notes at the beginning of the file, because that would lead to most masses being double counted as individual masses and as part of the planetary barycenters. When I download the newest version of that file and add up the masses for ids >= 10 I get 132890518795.53569 km^3/s^2, and if I use horizons to calculate osculating elements for a comet with mass "n.a." orbiting the SSB it uses a slightly larger GM value of 132890518817.90877, for a difference of 22.373 km^3/s^2.